# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/mochitest.ini
# Commit: 540c9159c3c6
# Full Hash: 540c9159c3c6dd350ac5343f9f9c977e4cdc53df
# Author: yulia <ystartsev@mozilla.com>
# Date: 2020-12-18 21:38:02
# Regressor Bug: 1681664
# File Overlap Count: 2
# Description:
#   Bug 1681664 - Allow Top-level await modules which fail to evaluate due to user action to fail without crashing. r=jonco,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99809
# ==============================================================================

diff -r 02e1e9b5931a -r 540c9159c3c6 js/xpconnect/tests/mochitest/mochitest.ini
--- a/js/xpconnect/tests/mochitest/mochitest.ini	Fri Dec 18 15:30:33 2020 +0000
+++ b/js/xpconnect/tests/mochitest/mochitest.ini	Fri Dec 18 12:37:36 2020 +0000
@@ -38,9 +38,11 @@
   test2_bug629331.html
   finalizationRegistry_worker.js
   private_field_worker.js
+  bug1681664_helper.js
 prefs =
   javascript.options.weakrefs=true
   javascript.options.experimental.private_fields=true
+  javascript.options.experimental.top_level_await=true
 
 [test_bug384632.html]
 [test_bug390488.html]
@@ -99,6 +101,7 @@
 [test_bug1094930.html]
 [test_bug1158558.html]
 [test_bug1448048.html]
+[test_bug1681664.html]
 [test_crosscompartment_weakmap.html]
 [test_enable_privilege.html]
 [test_frameWrapping.html]
@@ -123,4 +126,4 @@
 [test_weakRefs_cross_compartment.html]
 [test_weakRefs_collected_wrapper.html]
 [test_private_field_dom.html]
-[test_private_field_worker.html]
\ No newline at end of file
+[test_private_field_worker.html]